ch_pipeline/core/containers.py

```diff
@@ -942,6 +942,15 @@ def dataset_name_allowed(self, name: str) -> bool:
         parent_name, name = posixpath.split(name)
         return parent_name == "/" or self.group_name_allowed(parent_name)
 
+    @property
+    def time(self):
```

Comment:
Add a type annotation to this one.
```suggestion
    def time(self) -> np.ndarray:
```

Reply:
Added.